### PR TITLE
feat(promql): add language

### PR DESCRIPTION
**Pull request by MichaHoffmann**
- [x] I ran `make setup && make` to update the generated code after editing a `.atd` file (TODO: have a CI check)

### CODE REVIEW

**Comment by aryx** [comment]
I think you need to rebase on develop because you should not remove the unrelated .h extension that was added recently for C and C++.

File: lang.json [hunk 1]
```diff
@@ -48,8 +48,7 @@
     ],
     "maturity": "alpha",
     "exts": [
-      ".c",
```
Comment (aryx) [inline lang.json:51]: This should not be part of the PR.
You should probably rebase on main (or develop, whatever the master branch is).

**Review by aryx — CHANGES_REQUESTED** [review]

File: lang.json (hunk 1 above)
Comment (MichaHoffmann) [reply lang.json:51]: not sure what can be done; the commit that added those filetypes added them to this autogenerated file and i autogenerated it away. Ill at a commit to this PR to add them properly to `generate.py`

**Comment by aryx** [comment]
This also fix the error in https://github.com/returntocorp/semgrep-interfaces/pull/91

**Review by aryx — APPROVED** [review]
